index.html

```diff
@@ -1151,31 +1153,36 @@ <h3>Types</h3>
 
       <section>
         <h3>Issuer</h3>
+<p>
 
-        <p>
-<a>Issuer</a> information can be expressed using the following
-<a>properties</a>:
-        </p>
-
-        <dl>
-          <dt><var>issuer</var></dt>
-          <dd>
-The value of the <code>issuer</code> <a>property</a> MUST be a <a>URI</a>. It is
+The <a>Issuer</a> MUST be identified by using the
+<code>issuer</code> <a>property</a>. The value of the <code>issuer</code>
+<a>property</a> MUST be a <a>URI</a>. It is
 RECOMMENDED that dereferencing the <a>URI</a> results in a document containing
 machine-readable information about the <a>issuer</a> that can be used to
 <a>verify</a> the information expressed in the <a>credential</a>.
-          </dd>
-          <dt><var>issuanceDate</var></dt>
-          <dd>
+</p>
+</section>
+
+ <section>
+        <h3>Issuance Date</h3>
+<p>
+
+If present in a verifiable credential, then the date that the verifiable credential
+becomes valid MUST be identified by
+using the <code>issuanceDate</code> <a>property</a>.
 The value of the <code>issuanceDate</code> <a>property</a> MUST be a string
 value of an [[!ISO8601]] combined date and time string representing the date
```

Comment:
Are we mandating UTC time  or can any time zone be used?

Reply:
Any timezone can be used. It would be nice to mandate it, but developers will screw that up. We could make it a SHOULD, but that doesn't really buy us much (as people will ignore it, or not know about it).